packages/analyticsdx-template-lint/src/schemas/layout-schema.json

```diff
@@ -0,0 +1,442 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "http://schemas.salesforce.com/analytics/templates/ui.json",
+  "title": "Salesforce Analytics Template Layout Definition",
+  "description": "",
+  "type": "object",
+  "additionalProperties": false,
+  "allowComments": true,
+  "required": [],
+  "properties": {
+    "pages": {
+      "type": "array",
+      "minItems": 0,
+      "description": "",
+      "items": {
+        "type": "object",
+        "required": ["title", "layout"],
+        "additionalProperties": false,
+        "properties": {
+          "title": {
+            "type": "string",
+            "description": "Title displayed at top of the page."
```

Comment:
@mblumreich If you feel adventurous, all of the `description`s in here will show as hover text in the vscode editor. I copied some of them from the corresponding ui.json descriptions, but had to make up a few.